app/web/src/features/auth/jail/Jail.tsx

```diff
@@ -52,6 +53,7 @@ export default function Jail() {
   return (
     <>
       {!isJailed && <Redirect to="/" />}
+      <HtmlMeta title="More information required" />
       <PageTitle>More information required</PageTitle>
```

Comment:
Can we move these string values into constants as well please?

Reply:
done